projects/swimlane/ngx-charts/src/lib/area-chart/area-chart-normalized.component.ts

```diff
@@ -16,7 +16,7 @@ import { calculateViewDimensions } from '../common/view-dimensions.helper';
 import { ColorHelper } from '../common/color.helper';
 import { BaseChartComponent } from '../common/base-chart.component';
 import { id } from '../utils/id';
-import { getUniqueXDomainValues, getScaleType } from '../common/domain.helper';
+import { getScaleType, getUniqueXDomainValues, getXDomainArray } from '../common/domain.helper';
```

Comment:
Looks like `getXDomainArray` isn't being used. Same applies on the rest of the files in '/area-chart/'